depthai_bridge/src/ImuConverter.cpp

```diff
@@ -28,6 +28,28 @@ ImuConverter::ImuConverter(const std::string& frameName,
 
 ImuConverter::~ImuConverter() = default;
 
+void ImuConverter::updateRosBaseTime()
+{
+    ::ros::Time currentRosTime = ::ros::Time::now();
```

Comment:
I realize this I'm late an this has been merged but maybe this could work as a function in depthai uitility header to reduce repeating? https://github.com/Greenzie/depthai-ros/blob/noetic/depthai_bridge/include/depthai_bridge/depthaiUtility.hpp#L45